k8s/namespaces/lau/lau-case-frontend/lau-case-frontend.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: helm.fluxcd.io/v1
+kind: HelmRelease
+metadata:
+  name: lau-case-frontend
+spec:
+  releaseName: lau-case-frontend
+  chart:
+    git: git@github.com:hmcts/hmcts-charts
+    ref: master
+    path: stable/lau-case-frontend
+  values:
+    nodejs:
+      memoryRequests: "1024Mi"
```

Comment:
is this needed? frontend apps normally don't need to override the defaults unless they are large projects

Reply:
I've removed these overrides. Copied what we did with PCQ as the apps are of similar size.